api/handlers.go

```diff
@@ -0,0 +1,156 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// GoCertRouter takes in an environment struct, passes it along to any handlers that will need
+// access to it, then builds and returns it for a server to consume
+func GoCertRouter(env *Environment) http.Handler {
```

Comment:
Small nitpick, should this be named `NewGoCertRouter`?

Reply:
makes sense